server/src/modules/profile/util.service.ts

```diff
@@ -5,10 +5,25 @@ import { FilesRepository } from '@modules/files/repository';
 import { File } from '@entities/file.entity';
 import { IProfileUtilService } from '@modules/profile/interfaces/IService';
 import { CreateFileDto } from '@modules/files/dto/index';
+import { ProfileUpdateDto } from './dto';
+import { RequestContext } from '@modules/request-context/service';
+import { AUDIT_LOGS_REQUEST_CONTEXT_KEY } from '@modules/app/constants';
+import { User } from '@entities/user.entity';
 
 @Injectable()
 export class ProfileUtilService implements IProfileUtilService {
-  constructor(protected readonly filesRepository: FilesRepository, protected userRepository: UserRepository) {}
+  constructor(protected readonly filesRepository: FilesRepository, protected readonly userRepository: UserRepository) {}
+
+  private async setAuditLogForUser(user: User, resourceData?: any): Promise<void> {
```

Comment:
try to add audit logs call on services, if the function on util service is used by any other service, it will write audit logs